controllers/comments.go

```diff
@@ -23,20 +23,31 @@ func NewCommentController(db databases.Database, logger *zap.Logger) *commentCon
 	}
 }
 
+func (cc *commentController) LogInfo(message string) {
+	if cc.logger != nil {
+		cc.logger.Info(message)
+	}
+}
+
```

Comment:
doesn't need extra methods, nil check, and logger injection.
Logger is one of the few use cases that you can use as a singleton and global variable and people agree with you!

And the reason is because of simplicity and one logger object writing to one file, etc.

Reply:
fixed